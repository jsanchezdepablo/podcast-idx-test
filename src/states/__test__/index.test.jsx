import React from "react";
import { shallow } from "enzyme";
import GlobalStateProvider from "../";

const MOCK_PROPS = {
  children: <div className="test">Children</div>,
};

describe("States Component", () => {
  it("Matches snapshot", () => {
    const wrapper = shallow(<GlobalStateProvider {...MOCK_PROPS} />);
    expect(wrapper).toMatchSnapshot();
  });

  it("Should render children", () => {
    const wrapper = shallow(<GlobalStateProvider {...MOCK_PROPS} />);
    expect(wrapper.find(".test").exists()).toBeTruthy();
  });
});
