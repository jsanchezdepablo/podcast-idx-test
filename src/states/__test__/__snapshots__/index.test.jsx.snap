// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`States Component Matches snapshot 1`] = `
<Provider>
  <div
    className="test"
  >
    Children
  </div>
</Provider>
`;
